optaplanner-test/pom.xml

```diff
@@ -64,6 +64,24 @@
         <groupId>org.revapi</groupId>
         <artifactId>revapi-maven-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
```

Comment:
Is this temporary or permanent?

Reply:
Let's add an expirary comment, something like add "Remove when we depend on drools directly" comment - or even better, with jira url to that task

Reply:
=> will be done in next pr anyway apparently, good to leave as is